storage/influxdb/influxdb.go

```diff
@@ -163,6 +163,7 @@ func (self *influxdbStorage) tagPoints(ref info.ContainerReference, stats *info.
 		// merge with existing tags if any
 		addTagsToPoint(points[i], commonTags)
 		addTagsToPoint(points[i], ref.Labels)
+		addTagsToPoint(points[i], ref.Envs)
```

Comment:
What happens if there's overlap in env & label & common tag names? We work around this in the prometheus plugin by prefiixing the keys: [metrics/prometheus.go](https://github.com/google/cadvisor/blob/5e78498404b49fae5bd0e073a44f9f8d875a6778/metrics/prometheus.go#L568)